### PR TITLE
Fixes economy time

### PR DESCRIPTION
**Pull request by RyanSmake**
Until now ATMs and accounts randomly generated a time of the year every shift. Now they actually show our current date but with 544 years later.

🆑 Bxil
fix: Economy accounts now use the correct date.
/🆑

### CODE REVIEW

**Comment by taukausanake** [comment]
Question for admins/lore masters out there: So what happens to the year when we turn over a year in real life? Does Spess time progress the same as our time?
Is that a potential reason it was random to begin with maybe?

**Comment by RyanSmake** [comment]
You misunderstand: there was already a code which progressed time. ATMs did not use it. All I did is replace the random generator with the already existing year variable.

**Comment by SamHPurp** [comment]
@taukausanake last time that question was asked, I believe Fox said we're perpetually in this year, it doesn't tick over. Think Groundhog day.

**Comment by Shadeykins** [comment]
@taukausanake @SamHPurp 
Officially you're playing the same shift over and over again. This was the exact reason given when Fox implemented the "Timewarp.ogg" as an additional end-round sound.

A rolling timeline creates all sorts of issues for lore moving forward, mostly that it has to be created in perpetuity and that any relevant dates on wiki pages need to be updated annually. Unfortunately, despite the lack of mechanical precedent, there is no agreement on what the actual date really is. Several persons have been editing the wiki lore page unannounced to bump the time forward annually, while others are adamant that it remains a fixed date for sanity reasons.

tl;dr until the headmins actually make a firm decision (as the Lore Team has deferred to the headmins on this) or the maintainers step in and decide "ATM's should show a fixed date", you can presently assume it's a rolling date.

What happens with this PR will set the mechanical precedent that hopefully settles this issue.

**Comment by taukausanake** [comment]
> this shift, for all intents and purposes, is the latest event in the universe

Too bad that's not the case with the player base, but that's just me adding salt to the issue. I don't care for a set date because I like the idea of a rolling date. Means your character actually has progress in life. Hell, I'd be more happy if the whole station was a science experiment kind of like the Red vs. Blue Blood Gulch Chronicles or West World (better examples are out there but I got nothing). That would at least act as an excuse to research the same things over while still having some memory residue from your past lives. Maybe someone should throw that out on #wiki-dev

**Comment by RyanSmake** [comment]
I think a rolling timeline would be beneficial, and I remember that in june probably it created confusion that ATMs said it was december. It does not affect gameplay that much, but we do celebrate christmas IC when it's chritmas OOC for example.

An alternative solution would be to not to have years go forward, but make months and days the current ones.

**Comment by Shadeykins** [comment]
Months/days were always assumed to be rolling.

The only real contention is whether or not years are rolling.

@taukausanake Characters have always been able to progress, the only contention is trying to resolve the issue of "if this isn't a timewarp, that means the station gets nuked 7 times a day". That's why it's a "repeat" shift.

**Comment by RyanSmake** [comment]
@Shadeykins All shifts are semi-canon already, as such, in both ways, the issue you are presenting is kinda not an actual issue.

The station getting nuked is never canon, you having a drink in the bar is probably 90% of the time is canon. Whether years go forward or not this is not affected.

**Comment by tigercat2000** [comment]
Considering this a consistency fix - This is using the `game_year` variable, which is there for a reason. It makes more sense than the random nature of the ATM now. Plus, if we decide that +544 years is a horrible bad choice or something, it's better if we have one variable to change to static anyways.
